solutions/ova/kubernetes/ova-solution.yaml

```diff
@@ -87,102 +76,103 @@ solution:
       hare:
         hax:
           resources:
             requests:
-              memory: 128Mi
-              cpu: 250m
+              memory: 256Mi
+              cpu: 400m
             limits:
-              memory: 2Gi
-              cpu: 1000m
+              memory: 768Mi
+              cpu: 800m
       data:
         motr:
           resources:
             requests:
               memory: 1Gi
-              cpu: 250m
+              cpu: 500m
             limits:
               memory: 2Gi
               cpu: 1000m
         confd:
           resources:
             requests:
-              memory: 128Mi
+              memory: 256Mi
               cpu: 250m
             limits:
               memory: 512Mi
-              cpu: 500m
+              cpu: 400m
       server:
         rgw:
           resources:
             requests:
-              memory: 128Mi
-              cpu: 250m
+              memory: 1Gi
+              cpu: 500m
             limits:
               memory: 2Gi
-              cpu: 2000m
+              cpu: 1500m
       control:
         agent:
           resources:
             requests:
               memory: 128Mi
-              cpu: 250m
+              cpu: 200m
             limits:
               memory: 256Mi
-              cpu: 500m
+              cpu: 250m
       ha:
         fault_tolerance:
           resources:
             requests:
               memory: 128Mi
-              cpu: 250m
+              cpu: 200m
             limits:
-              memory: 1Gi
-              cpu: 500m
+              memory: 512Mi
+              cpu: 250m
         health_monitor:
           resources:
             requests:
               memory: 128Mi
-              cpu: 250m
+              cpu: 200m
             limits:
-              memory: 1Gi
-              cpu: 500m
+              memory: 512Mi
+              cpu: 250m
         k8s_monitor:
           resources:
             requests:
               memory: 128Mi
-              cpu: 250m
+              cpu: 200m
             limits:
-              memory: 1Gi
-              cpu: 500m
-  storage:
-    cvg1:
-      name: cvg-01
-      type: ios
-      devices:
-        metadata:
-          device: /dev/sdc
-          size: 5Gi
-        data:
-          d1:
-            device: /dev/sdd
-            size: 5Gi
-          d2:
-            device: /dev/sde
-            size: 5Gi
-
-  nodes:
-    node1:
-      name: cortx-ova-rgw.seagate.com
+              memory: 512Mi
+              cpu: 250m
+  storage_sets:
+    - name: storage-set-1
+      durability:
+        sns: 1+0+0
+        dix: 1+0+0
+      container_group_size: "1"
+      storage:
+        - name: cvg-01
+          type: ios
+          devices:
+            metadata:
+              - path: /dev/sdb
+                size: 5Gi
+            data:
+              - path: /dev/sdc
+                size: 5Gi
+              - path: /dev/sdd
+                size: 5Gi
+      nodes:
+        - cortx-ova-rgw.seagate.com
```

Comment:
can we rename hostname to `cortx-rgw-ova.seagate.com`

Reply:
Actually we should remove `rgw` now